pkg/sync/v6/client.go

```diff
@@ -254,3 +259,24 @@ func printDiff(log *logrus.Entry, existing, o *unstructured.Unstructured) bool {
 	}
 	return diffShown
 }
+
+// reconcileProtected check if object is marked with azure reconcile protection annotation
+// this will remove certain objects from being updated post cluster creation
+// access to modify these objects is limited by RBAC
+func (s *sync) reconcileProtected(existing, o *unstructured.Unstructured) bool {
+	if o.GetAnnotations()[reconcileProtectKeyAnnotationKey] != "" {
```

Comment:
If the value is not null but e.g. "false", this will still kick in.. do we want to explicitly check that it's true?

Reply:
done